Core/Importers/UkrSibTransactionsImporter.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Core.Importers
+{
+    class UkrSibTransactionsImporter : TransactionsImporter
+    {
+        public override List<Transaction> Load(Stream file)
+        {
+            throw new NotImplementedException();
```

Comment:
Codacy found an issue: [Implement this method or throw 'NotSupportedException' instead.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5836065)